website/blog/2024-06-21-AgentEval/index.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "AgentEval: A Developer Tool to Assess Utility of LLM-powered Applications"
+authors:
+  - jluey
+  - julianakiseleva 
+tags: [LLM, GPT, evaluation, task utility] 
+---
+ 
+
+![Fig.1: An AgentEval framework with verification step](img/agenteval_ov_v3.png) 
+
+<p align="center"><em>Fig.1 illustrates the general flow of AgentEval with verification step </em></p> 
+
+ 
+
+TL;DR: 
+* As a developer of an LLM-powered application, how can you assess the utility it brings to end users while helping them with their tasks? 
```

Comment:
The term "utility" is vague. Any other suggestions?

Reply:
maybe "* As a developer, how can you assess the utility and effectiveness of an LLM-powered application in helping end users with their tasks? "

Reply:
@BeibinLi  we used and defined in the previous post https://microsoft.github.io/autogen/blog/2023/11/20/AgentEval/, and the paper as well. We can repeat it here if it would be more clear

Reply:
@jluey1  Yes, " utility and effectiveness of an LLM-powered application" feels better. That correction would improve the readability of this line.